source/post.mysimulation.cfg

```diff
@@ -1,58 +1,114 @@
 [default]
-input = /lcrc/group/e3sm/ac.golaz/E3SM_simulations/20210305.v2beta3.piControl.ne30pg2_EC30to60E2r2.chrysalis
-output = /lcrc/group/e3sm/ac.golaz/E3SM_simulations/20210305.v2beta3.piControl.ne30pg2_EC30to60E2r2.chrysalis
-case = 20210305.v2beta3.piControl.ne30pg2_EC30to60E2r2.chrysalis
-www = /lcrc/group/e3sm/public_html/diagnostic_output/ac.golaz/E3SM/v2/beta
+# The directory with the data to use
```

Comment:
Should I add comments for every single parameter in the configuration file? Alternatively, should the parameters be defined on another page of the documentation?

Reply:
It might be good to at least say what parameters are required for each set in E3SM Diags (e.g., which sets require time series parameters), as this was an issue in #71.

Reply:
Added parameter descriptions. More can be added in #78.